content/en/event-driven_architecture.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Event-Driven Architecture
+status: Feedback Appreciated
+category: concept
+---
+## What it is
+Event-driven architecture is a software architecture that promotes the creation, processing, and consumption of events. 
+An event is any change to an application's state.
+For example, hailing a ride on a ride sharing app represents an event.
+This architecture creates the structure in which events can be properly routed from their source (my app requesting that ride) to the desired receivers (the apps of available drivers nearby).
+## Problem it addresses
+As more data becomes real-time, finding reliable ways to ensure that events are captured and routed to the appropriate [service](https://glossary.cncf.io/service/) that must process event requests gets increasingly challenging. 
+Many applications need a way to guarantee event delivery to ensure the application functions as designed.
+With multiple event sources and event receivers, it also becomes challenging to ensure that events are properly routed to their intended recipient services.
+## How it helps
```

Comment:
How it helps is still a little on the jargon-y side. I think talk of producers and consumers should either be omitted or referred to in the What it is section. IMO event driven architecture helps by allowing organizations/developers to focus on reacting to events and save on compute costs by reducing the number of app instances that need to exist during low traffic periods. 

I think the entry is in great shape overall and I'm happy to set up a pairing session to finish it if that would be helpful. Thanks a ton for your work here.

Reply:
Made some changes